book/projects/project_roadmap.md

```diff
@@ -44,10 +44,11 @@ We facilitate a project team formation session at the end of the first day of th
 * project work begins and teams have regular check-ins to chart progress
 * project outputs are recorded in a GitHub repository
 
-## **Presentations** 
+## **Deliverables** 
+
+* each team is requested to create a project readme using [this template](project_readmes/project_readme_template) and contribute it to the this Jupyterbook. Please check in with a helper if your team would like guidance through this process. The basic workflow will require creating a branch, copying the template file, filling in the template file and adding it to the list of projects, committing your changes, opening a pull request (PR), and reviewing and merging your changes.

```

Comment:
For consistency, want to capitalize the beginning of each bullet point?

Reply:
Consistency above (lower case, clearly the one I looked at) or below (upper case)?

Reply:
My vote is upper case :shrug: